appveyor.yml

```diff
@@ -13,8 +13,18 @@ init:
 - ps: New-Item -Type HardLink -Path "C:\Python37\python3.exe" -Value "C:\Python37\python.exe"
 
 build_script:
-- '"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvarsx86_amd64.bat"'
+- '"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars64.bat"'
 - msbuild C:\wnbd\vstudio\wnbd.sln /property:Configuration=%CONFIGURATION%
+- ps: |
+    if ($env:CONFIGURATION -eq "Release") {
+      pushd
+      cd C:\wnbd\vstudio\;
+      msbuild wnbd.sln /property:Configuration=Release
```

Comment:
Wouldn't it be already be built at line 17? Should we do it for "Analysis" builds instead since those are already doing some static code analysis?